app/model/ContentItem.scala

```diff
@@ -2,37 +2,27 @@ package model
 
 import ai.x.play.json.Jsonx
 import com.amazonaws.services.dynamodbv2.document.Item
-import play.api.Logger
+import cats.syntax.either._
+import model.command.{CampaignCentralApiError, JsonParsingError}
 import play.api.libs.json.{Format, JsValue, Json}
 
-import scala.util.control.NonFatal
+case class ContentItem(campaignId: String,
+                       id: String,
+                       `type`: String,
+                       composerId: Option[String],
+                       path: Option[String],
+                       title: String,
+                       isLive: Boolean) {
 
-case class ContentItem(
-  campaignId: String,
-  id: String,
-  `type`: String,
-  composerId: Option[String],
-  path: Option[String],
-  title: String,
-  isLive: Boolean,
-  atoms: List[Atom] = Nil
```

Comment:
What was `atoms` used for originally I wonder?

Reply:
In hosted pages the CTA is an atom, and in hosted video pages the video component is an atom.